stand/efi/loader/arch/amd64/Makefile.inc

```diff
@@ -1,15 +1,40 @@
 
+.PATH: ${BOOTSRC}/i386/libi386
+SRCS+=	nullconsole.c \
+	comconsole.c \
+	spinconsole.c
+
+CFLAGS+=	-fPIC
+LDFLAGS+=	-Wl,-znocombreloc
+
+.if ${DO32:U0} == 0
 SRCS+=	amd64_tramp.S \
 	start.S \
 	elf64_freebsd.c \
 	trap.c \
 	multiboot2.c \
 	exc.S
 
-.PATH:	${BOOTSRC}/i386/libi386
-SRCS+=	nullconsole.c \
-	comconsole.c \
-	spinconsole.c
+SUBDIR.${MK_LOADER_IA32}+=	../loader_${LOADER_INTERP}-ia32
```

Comment:
Why isn't this up in stand/efi/Makefile? That would be the normal way to do this

Reply:
because different architectures have different consoles.
comconsole.c is x86 only, for example, and is a mistake from the past that bypasses UEFI entirely to allow better configurability of the serial console than UEFI can do w/o us adding ACPI parsing to it (so it's the least bad mistake we can likely make given the x86 expectations from the users).

Reply:
No I mean the SUBDIR bit; that's where loader_lua etc get recursed into from, so should also be where loader_lua-ia32 etc are recursed into?

Reply:
> Why isn't this up in stand/efi/Makefile? That would be the normal way to do this

I thought this would be more convenient since we could just use `LOADER_INTERP`, but now I realize that `SUBDIR.yes.yes` is a thing so we can do `SUBDIR.${MK_FORTH}.${MK_LOADER_IA32}`. Will update.

> Oh that. Yes. In fact, I think we likely just want loader_ia32.efi and not all the variations since it's a special thing.

Hm, so would that only build the lua loader?